helpers/lib/fonts.js

```diff
@@ -18,34 +16,29 @@ const fontProviders = {
          * @returns {string}
          */
         parser: function(fonts) {
-            var collection = [],
-                familyHash = {};
+            const collection = [];
+            const familyHash = {};
 
-            _.each(fonts, function fontsIterator(font) {
-                var split = font.split('_'),
-                    familyKey = split[1],  // Eg: Open+Sans
-                    weights = split[2];    // Eg: 400,700italic
+            fonts.forEach(font => {
```

Comment:
I think forEach is not going to give us a performance benefit. We should use a plain for loop: https://medium.com/tech-tajawal/loops-performances-in-node-js-9fbccf2d6aa6

Similarly for any other usages of forEach.

Reply:
fixed.